src/content/docs/en/pages/secure-journey/intelligent-dns-advanced-configurations/add-lets-encrypt-txt-record.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: How to add a TXT record to configure Let's Encrypt certificate
+description: After you create a Let’s Encrypt certificate in a platform other than Azion, but want to use it with a zone created in Intelligent DNS, you must also create TXT record to add the information.
```

Comment:
Me parece que a descrição está deslocada do título.

Que tal:
`When creating a Let's Encrypt certificate externally for a hostname that is hosted in Intelligent DNS you must be able to add the TXT record required for the DNS challenge. Here you'll find how to do it.`